miden-lib/asm/sat/kernel.masm

```diff
@@ -258,6 +271,9 @@ end
 #!
 #! - ASSET is the asset to remove from the vault.
 export.account_vault_remove_asset
+    # emit event to signal that an asset is being removed from the account vault
+    push.1 drop emit.REMOVE_ASSET_FROM_ACCOUNT_VAULT_EVENT
```

Comment:
I would add a `TODO` comment (with link to the issue) explaining why we do `push.1 drop`.